### PR TITLE
SVDB MERGE update

### PR DESCRIPTION
**Pull request by WackerO**
Updated the SVDB MERGE module

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/modules/svdb/merge/main.nf [hunk 1]
```diff
@@ -37,6 +36,7 @@ process SVDB_MERGE {
         $prio \\
         --vcf $input \\
         > ${prefix}_sv_merge.vcf
+    bgzip ${prefix}_sv_merge.vcf
```
Comment (SusiJo) [inline modules/nf-core/modules/svdb/merge/main.nf:40]: Ah just saw that you're not adding the samtools version to the `versions.yml`

**Review by SusiJo — COMMENTED** [review]
Please ask if you need to add the samtools version before merging.

File: conf/modules.config [hunk 2]
```diff
@@ -1104,8 +1104,8 @@ process{
             ext.prefix       = { "${meta.id}.tiddit" }
             publishDir       = [
                 mode: params.publish_dir_mode,
-                path: { "${params.outdir}/variant_calling/tiddit/${meta.id}/" },
-                pattern: "*{vcf,vcf.gz}"
+                path: { "${params.outdir}/variant_calling/${meta.id}/tiddit" },
```
Comment (maxulysse) [inline conf/modules.config:1107]:
```suggestion
                path: { "${params.outdir}/variant_calling/tiddit/${meta.id}" },
```

File: modules/nf-core/modules/svdb/merge/main.nf (hunk 1 above)
Comment (maxulysse) [reply modules/nf-core/modules/svdb/merge/main.nf:40]: Can you fix that @WackerO ?

**Comment by FriederikeHanssen** [comment]
yep i am with susi, the modules should be updated to also add the samtools version

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
I'm assuming that the expected file is no longer a `vcf`, but a `vcf.gz`.
Can you update the test @WackerO ?

**Review by SusiJo — APPROVED** [review]
Looking good! Let's just wait for the tests to pass

**Pull request merged**